### PR TITLE
HDDS-9231. Recon: Heatmap Page can be directly accessed even after disable config

### PR DESCRIPTION
**Pull request by smitajoshi12**
## What changes were proposed in this pull request?
Hetamap can be accessed after disabling config flag. so addressed this bug with this PR.

## What is the link to the Apache JIRA
https://issues.apache.org/jira/browse/HDDS-9231

## How was this patch tested?

Manually

image

### CODE REVIEW

**Comment by umamaheswararao** [comment]
@devmadhuu could you please take a look at it?

File: hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/components/navBar/navBar.tsx [hunk 1]
```diff
@@ -71,14 +72,9 @@ class NavBar extends React.Component<INavBarProps> {
...
 
-  refresh = () => {
-    console.log("refresh");
-    this.props.history.push('/Heatmap');
-    window.location.reload();
-    }
-
   render() {
     const {location} = this.props;
+    const { isHeatmapAvailable } = this.state;
```
Comment (devmadhuu) [inline hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/components/navBar/navBar.tsx:77]: @smitajoshi12 thanks for working on this patch. However can we keep the variable naming consistent. Here we have defined as "`isHeatmapAvailable`" and in heatmap.tsx file, we have defined as `isHeatmapEnabled`

**Review by devmadhuu — COMMENTED** [review]
LGTM , except one minor comment. @devpalabhishek , can you have a look ?

File: hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/components/navBar/navBar.tsx (hunk 1 above)
Comment (smitajoshi12) [reply hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/components/navBar/navBar.tsx]: @devmadhuu 
updated variable name as per suggestion.

**Review by devmadhuu — APPROVED** [review]
Thanks @smitajoshi12 LGTM +1

**Review by sumitagrawl — APPROVED** [review]
@smitajoshi12 Thanks for working over this, LGTM +1

**Pull request merged**